Adding azure functions docs to the python and node integrations (#1856)
getsentry/sentry-docs: src/collections/_documentation/platforms/node/azure_functions.md

---
title: Azure Functions
sidebar_order: 9
---

> Install our Nodejs SDK in Azure:

Reviewer: ```suggestion
Install our Node.js SDK in Azure:
```

> ```basic
>  npm install @sentry/node
> ```

Reviewer: ```suggestion
```bash
npm install @sentry/node
```
```

Set up Sentry error logging for your azure function:

> ```jsx
> const Sentry = require("@sentry/node");
> 
> Sentry.init({
>   dsn: "<Your dsn>",
> });
> 
> module.exports = async function (context, req) {
>     try {
>         callUndefinedFunction(); // Call undefined function.
>     } catch (error) {
>         Sentry.captureException(error); // Capture the exception in Sentry dashboard.
>         await Sentry.flush(2000);
>     }
> 
>     context.res = {
>         status: 200, /* Defaults to 200 */
>         body: "Hello from Azure cloud function"
>     };
> }
> ```

Reviewer: ```suggestion
```js
const Sentry = require("@sentry/node");

Sentry.init({
    dsn: "___PUBLIC_DSN___"
});

module.exports = async function (context, req) {
    try {
        await callUndefinedFunction(); // Call undefined function.
    } catch (error) {
        Sentry.captureException(error); // Capture the exception in Sentry dashboard.
        await Sentry.flush(2000);
    }

    context.res = {
        status: 200, /* Defaults to 200 */
        body: "Hello from Azure cloud function"
    };
}
```
```

You can obtain the DSN using your Sentry account from your organization's *Settings -> Projects -> Client Keys (DSN)* in the Sentry web UI.

> Note: You need to call both `captureException` and `flush` to report errors into Sentry.

Reviewer: ```suggestion
Note: You need to call both `captureException` and `flush` in order for captured events to be successfully delivered to Sentry.
```

---
title: Azure Functions
sidebar_order: 9
---

Install our Nodejs SDK in Azure:
 ````js
 npm install @sentry/node
 ````

Set up Sentry error logging for your azure function:
````jsx
const Sentry = require("@sentry/node");

Sentry.init({
  dsn: "<Your dsn>",
});

module.exports = async function (context, req) {

>     try{
>         callUndefinedFunction();  // Call undefined function.
>     }catch(error){
>         Sentry.captureException(error);  // Capture the exception in Sentry dashboard.

Reviewer: ```suggestion
    try {
        callUndefinedFunction(); // Call undefined function.
    } catch (error) {
        Sentry.captureException(error); // Capture the exception in Sentry dashboard.
```

>         Sentry.flush(2000);

Reviewer: ```suggestion
        await Sentry.flush(2000);
```

    }

    context.res = {
        status: 200, /* Defaults to 200 */
        body: "Hello from Azure cloud function"
    };
}
````

You can obtain the DSN using your Sentry account from your organization's *Settings -> Projects -> Client Keys (DSN)* in the Sentry web UI.

> Note: You need to call both `CaptureException` and `Flush` to report errors into Sentry.

Reviewer: ```suggestion
Note: You need to call both `captureException` and `flush` to report errors into Sentry.
```
